Storage/Storage/SQL/VisitsTable.swift

```diff
@@ -0,0 +1,53 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+let TableNameVisits = "visits"
+
+class VisitsTable<T>: GenericTable<Visit> {
+    override var name: String { return TableNameVisits }
+    override var rows: String { return "siteGuid TEXT NOT NULL, " +
+                                       "date REAL NOT NULL" }
```

Comment:
Any particular reason to use a REAL rather than NUMBER?

Also might be worth calling the column `date_msec` (if that's what it is).